src/main/java/core/basesyntax/strategy/impl/ReturnOperationHandler.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+
+public class ReturnOperationHandler implements OperationHandler {
+    @Override
+    public void handleTransaction(FruitTransaction transaction) {
+        String key = transaction.getFruit();
+        if (!Storage.DATA_BASE.containsKey(key)) {
+            //just some logic
+            throw new RuntimeException("No such fruit in shop");
+        }

```

Comment:
i don't sure that we need this check, because we can just use Storage.DATA_BASE.getOrDefault(key, 0). But up to you